deploy/security_context_constraints.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: security.openshift.io/v1
```

Comment:
I would rename the file to scc.yaml - I wouldn't expect to find it with this full name and might miss it